src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * The main service provider for the version 2 of the Widgets.
+ *
+ * @since   TBD
+
```

Comment:
:no_entry_sign: There must be a single blank line after a tag group

<sub>hash: a81e37ad27cbf24059afc81e31a2db6a</sub>